viewer/packages/api/src/public/RevealManager.ts

```diff
@@ -106,13 +106,14 @@ export class RevealManager {
       !this._lastCamera.position.equals(camera.position) ||
       !this._lastCamera.quaternion.equals(camera.quaternion);
 
+    this._cadManager.updateCamera(camera, hasCameraChanged);
```

Comment:
This will trigger the sector culler on every frame. Any reason you moved it here?

Reply:
When the camera pose is equal to that of the previous frame, the `CadManager.updateCamera` function will be called with hasCameraChange equal to false. This first invocation will trigger the sector culling, but subsequent calls will be filtered out (due to the `distinctUntilKeyChanged` RxJS operator in `CadModelUpdateHandler`. 

One could of course do the 'is this the first frame of the camera not moving'-logic in `RevealManager`. I opted to keep things in `CadModelUpdateHandler`, but I'm more than happy to move it if you'd like.

Reply:
But why should we trigger an invocation if the camera hasn't changed? If f.ex. the camera doesn't move at all we don't want to drain resources here triggering sector culling.

Reply:
It gets filtered out, yes